core-aam/aria-keyshortcuts-manual.html

```diff
@@ -24,6 +24,14 @@
                   "keyshortcuts:Shift+Space"
                ]
             ],
+            "AXAPI" : [
+               [
+                  "property",
+                  "AXKeyShortcutsValue",
+                  "is",
+                  "Shift+Space"
+               ]
```

Comment:
```suggestion
               [
                  "property",
                  "AXCustomContent",
                  "contains",
                  "{ label: \"shortcut\", value: \"Shift+Space\" }"
               ]
```
Need to verify, but more likely this.

Reply:
Note: Don't commit that diff yet, pls. Thx.